lang/mr.json

```diff
@@ -0,0 +1,98 @@
+{
+    "Audio Player": "ध्वनि वादक",
+    "Video Player": "चित्रवाणी वादक",
+    "Play": "वाजवा",
+    "Pause": "थांबा",
+    "Replay": "पुन्हा वाजवा",
+    "Current Time": "वर्तमान वेळ",
+    "Duration": "कालावधी",
+    "Remaining Time": "उर्वरित वेळ",
+    "Stream Type": "प्रवाह प्रकार",
+    "LIVE": "थेट प्रसारण",
+    "Seek to live, currently behind live": "थेट प्रसारणकडे जा, चालू क्षणाला थेट प्रसारण पासुन मागे आहे",
+    "Seek to live, currently playing live": "थेट प्रसारणकडे जा, चालू क्षणाला थेट प्रसारण आहे",
+    "Loaded": "सज्ज",
+    "Progress": "प्रगती",
+    "Progress Bar": "प्रगती पट्टा",
+    "progress bar timing: currentTime={1} duration={2}": "प्रगती पट्टा वेळ: {1} of {2}",
```

Comment:
This one shouldn't be a literal translation of the key. Instead, the value should be just a translation of "{1} of {2}". Compare the Hindi translation:

https://github.com/videojs/video.js/blob/a57b07a418636daf9723a28715f96f2e3152ddc3/lang/hi.json#L17